src/components/device-page/states.tsx

```diff
@@ -36,7 +36,8 @@ const readonlyFields = [
     "elapsed",
     "contact",
     "action",
-    "click"
+    "click",
+    "update_available"
```

Comment:
I believe we want to add the new field too? `update` ?

Reply:
Yep, hit the Edit button:)